hack/quick-install.sh

```diff
@@ -15,35 +15,28 @@ kubectl apply -f https://github.com/knative/eventing/releases/download/knative-$
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
-kubectl apply -f - << EOF
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: config-features
-  namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
-    knative.dev/config-propagation: original
-    knative.dev/config-category: eventing
-data:
-  new-apiserversource-filters: enabled
-EOF
-
 bash cmd/operator/generate.sh
 helm template kubearchive charts/kubearchive -n kubearchive \
     --include-crds \
     --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
 ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > /tmp/kubearchive.yaml
 kubectl apply -n kubearchive -f /tmp/kubearchive.yaml
 
-cat << EOF > /tmp/patch.yaml
-stringData:
-  POSTGRES_PORT: "5432"
-  POSTGRES_URL: postgresql.databases.svc.cluster.local
-  POSTGRES_USER: kubearchive
-  POSTGRES_DB: kubearchive
-  POSTGRES_PASSWORD: 'P0stgr3sdbP@ssword'  # notsecret
-EOF
-kubectl patch -n kubearchive secrets kubearchive-database-credentials --patch-file /tmp/patch.yaml
-kubectl rollout restart deployment --namespace=kubearchive kubearchive-sink kubearchive-api-server
```

Comment:
I think this should change and we should leave this out.  We should use our development credentials for everything, so things work right out-of-the-box.  And we should tell users what to change to make it production ready.

We can have a test case that does this, but I don't see any advantage to having settings that don't work at all.

Reply:
> We can have a test case that does this, but I don't see any advantage to having settings that don't work at all.

How can the settings not work?
IIUC, we can do one of the following:

* Follow the `DEVELOPMENT.md` instructions to set up the development environment, i.e., run the `install.sh` for deploying the database and install the kubearchive helm chart in non-production mode. This will work as intended.

* Run the `quick-install.sh` script to deploy everything from scratch. When this script install the helm chart it's done with the setting `global.production=true`. With the current chart, this means the secret will be deployed without credentials because it expects the user to provide them with a custom one, then, in the script, we set this credentials as we expect the user to do.

So, both of them, either running quick-install or going through the development process will end up with settings that are expected to work.

If we test the "production-like" deployment here by changing the db credentials on kubearchive here, I think we should then add a specific integration test for doing it.

Reply:
> How can the settings not work?

The initial production settings do not work, they are all empty strings.  The user must change them first before they can do anything.

Reply:
Based on our discussion today I'm going to resolve this conversation.

I've created https://github.com/kubearchive/kubearchive/issues/442 to update the documentation.

Reply:
After the last talk, I am ok with going with your suggestion, specially considering that we will change the user instructions to modify the kubearchive YAML *before* applying it and not afterwards in https://github.com/kubearchive/kubearchive/issues/380.

Nevertheless, we could have an integration test for testing changes in the database credentials (like password rotation) that will need to have kubearchive application rolled out. I created an issue to address this: https://github.com/kubearchive/kubearchive/issues/444